apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -564,3 +468,159 @@ function DeleteQuestionModal({
     `,
   });
 }
+
+function questionTestsForm({
+  questionTestPath,
+  questionTestCsrfToken,
+}: {
+  questionTestPath: string;
+  questionTestCsrfToken: string;
+}) {
+  return html`
+    <form name="question-tests-form" method="POST" action="${questionTestPath}">
+      <input type="hidden" name="__csrf_token" value="${questionTestCsrfToken}" />
+      <button class="btn btn-sm btn-outline-primary" name="__action" value="test_once">
+        Test once with full details
+      </button>
+      <button class="btn btn-sm btn-outline-primary" name="__action" value="test_100">
+        Test 100 times with only results
+      </button>
+    </form>
+  `;
+}
+
+function QuestionSharing({
+  questionSharedPublicly,
+  sharingSetsIn,
+  hasCoursePermissionOwn,
+  sharingSetsOther,
+  csrfToken,
+  qid,
+}: {
+  questionSharedPublicly: boolean;
+  sharingSetsIn: SharingSet[];
+  hasCoursePermissionOwn: boolean;
+  sharingSetsOther: SharingSet[];
+  csrfToken: string;
+  qid: string;
+}) {
+  if (questionSharedPublicly) {
+    return html`
+      <div class="row px-3">
```

Comment:
Can you say more about why you added horizontal padding here and elsewhere?

Reply:
As stated above, I thought it improved readability by indenting and adding a little bit of lateral separation from the header. Removed in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348).